Update idioms.md with suggestion to use inline value classes (#4049)
JetBrains/kotlin-web-site: docs/topics/idioms.md

}
```

> ## Use inline value classes to wrap single values for type-safety

Reviewer: ```suggestion
## Use inline classes for type-safe values
```

```kotlin
@JvmInline
value class EmployeeId(private val id: String)

@JvmInline
value class CustomerId(private val id: String)
```

> Now accidentally mixing up `EmployeeId` and `CustomerId` will result in a compilation error. The inline classes will be represented by the 
> underlying type at runtime. `@JvmInline` is only needed for JVM backends.

Reviewer: ```suggestion
If you accidentally mix up `EmployeeId` and `CustomerId`, a compilation error is triggered.

> The `@JvmInline` annotation is only needed for JVM backends.
>
{type="note"}
```

> (See [Inline classes](inline-classes.md) for more information)

Reviewer: ```suggestion
For more information, see [Inline value classes](inline-classes.md).
```
